src/index.js

```diff
@@ -90,16 +90,59 @@ function cropScreenshot([rect, screenshot]) {
  * @returns {Object|undefined}
  */
 function getElementBoundingRect(elementSelector) {
-    var element = document.querySelectorAll(elementSelector)[0];
-    if (element) {
+    function computeFrameOffset(win, dims) {
```

Comment:
Thanks for the feedback. But I'm not sure what to do with this comment. The `execute` command converts the `getElementBoundingRect` function to a string to send to the browser so I cannot reference other functions inside of it. Do you have any suggestions?